irods/test/helpers.py

```diff
@@ -151,6 +151,18 @@ def recast(k):
 # Create a connection for test, based on ~/.irods environment by default.
 
 def make_session(test_server_version = True, **kwargs):
+    '''
+    Connect to an active iRODS server as determined by any client environment
+    file present at a standard location, and by any keyword arguments given.
+
+    Arguments:
+
+    test_server_version (True or False).  Raises iRODS_Server_Too_Recent if
```

Comment:
- Remove `(True or False)`
- State the expected data type in the parameter's description
- State the default value in the parameter's description

Reply:
Done